std/conv.d

```diff
@@ -2186,6 +2188,15 @@ Lerr:
     assertCTFEable!({ string s =  "1234abc"; assert(parse!uint(s) ==  1234 && s == "abc"); });
 }
 
+// Issue 13931
+@safe pure unittest
+{
+    import std.exception;
+
+    assertThrown!ConvOverflowException("-21474836480".to!int());
+    assertThrown!ConvOverflowException("-92233720368547758080".to!long());
```

Comment:
Nop. All cases the issue is about are listened. One can improve unittests if he wants but it will not be connected with this issue and this pull.